README.md

```diff
@@ -25,110 +36,188 @@ Head to [our playground](https://livekit.io/playground) and give it a spin. Buil
-server. This app is built with our [React SDK](https://github.com/livekit/livekit-react).
+## Getting Started
 
-Once connected, your video and audio are now published to your new LiveKit instance!
+### Starting LiveKit
 
-### Generating access tokens (JWT)
+Start LiveKit by running `livekit-server` with no arguments. It will start in development mode by default with a set of placeholder API key/secret pair.
 
-To add more users to a room, you'll have to create a token for each
-participant. [Learn more about access tokens](https://docs.livekit.io/guides/access-tokens/).
+```
+API Key: devkey
+API Secret: secret
+```
+
+To customize your setup for production, refer to [deployment docs](https://docs.livekit.io/deploy/)
 
-`livekit-server` provides a convenient sub-command to create a development token. This token has an expiration of a
-month, which is useful for development and testing, but not appropriate for production use.
+### Creating an Access Token
+
+Each user that connects to a LiveKit room requires an [Access Token](https://docs.livekit.io/guides/access-tokens/). It's a JWT token that encodes the user's identity as well as permissions granted to them. You can generate tokens with our CLI:
 
 ```shell
-docker run --rm -e LIVEKIT_KEYS="<api-key>: <api-secret>" \
-    livekit/livekit-server create-join-token \
-    --room "<room-name>" \
-    --identity "<participant-identity>"
+livekit-cli create-token \
+    --api-key devkey --api-secret secret \
+    --join --room my-first-room --identity user1 \
+    --valid-for 24h
 ```
 
-## Deploying to server
-
-Deployment Docs: https://docs.livekit.io/deploy/
+### Test with example app
 
-### Single node server
+Head over to the [example app](https://example.livekit.io) and enter the generated token to connect to your LiveKit
+server. This app is built with our [React SDK](https://github.com/livekit/livekit-react).
 
-Use our deploy config generator to set up a single node deployment with automatic TLS termination and built-in TURN.
+Once connected, your video and audio are now published to your new LiveKit instance!
 
-It includes a cloud-init/setup script that's supported by most cloud environments.
+### Simulating a test publisher
 
 ```shell
-docker run --rm -it -v$PWD:/output livekit/generate
+livekit-cli join-room \
+    --url ws://localhost:7880 \
+    --api-key devkey --api-secret secret \
+    --room my-first-room --identity bot-user1 \
+    --publish-demo
 ```
 
-### Kubernetes
-
-We publish a [helm chart](https://github.com/livekit/livekit-helm) that helps you to set up a cluster with high
-availability. For detailed instructions, see [Kubernetes guide](https://docs.livekit.io/deploy/kubernetes)
+This command publishes a demo video loop to the room. Because of the way the video clip is encoded (keyframes every 3 seconds), there is a slight delay before browser has sufficient data to start rendering frames. This is an artifact of the simulation.
```

Comment:
```suggestion
This command publishes a demo video loop to the room. Because of the way the video clip is encoded (keyframes every 3 seconds), there is a slight delay before the browser has sufficient data to start rendering frames. This is an artifact of the simulation.
```